src/main/webapp/basic-search.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+import Button from '@material-ui/core/Button'
+import Card from '@material-ui/core/Card'
+import CardContent from '@material-ui/core/CardContent'
+import TextField from '@material-ui/core/TextField'
+
+import { executeQuery } from './intrigue-api/lib/cache'
+
+const TextSearch = props => {
+  const [filterTree, setFilterTree] = React.useState({
+    type: 'ILIKE',
+    property: 'anyText',
+    value: '',
+  })
+
+  return (
+    <CardContent>
+      <TextField
+        label="Text"
+        variant="outlined"
+        fullWidth
+        value={filterTree.value}
+        onChange={e => {
+          const value = e.target.value
+          setFilterTree({ ...filterTree, value })
+        }}
+      />
+
+      <Button
+        fullWidth
+        variant="contained"
+        color="primary"
+        onClick={() => {
```

Comment:
✏️ Could be one line
```suggestion
        onClick={() => props.onSearch(filterTree)}

```

Reply:
Good catch. 
Changed